SpinDance_Fork_Changes.md

```diff
@@ -0,0 +1,175 @@
+
+A handy way to view the diff is via this [Github pull request link](https://github.com/espressif/esp-idf/compare/v5.0...spindance:esp-idf:spindance_changes_v5.0).
+
+The original ESP repo and this SpinDance fork can be cloned and viewed separately via:
+```
+git clone git@github.com:espressif/esp-idf.git esp-idf
+cd esp-idf
+git checkout v5.0
+
+cd ..
+git clone git@github.com:spindance/esp-idf.git spindance-esp-idf
+cd spindance-esp-idf
+git checkout spindance_changes_v5.0
+```
+
+## Changed Files
+```
+components/lwip/port/esp32/include/netif/wlanif.h
+components/lwip/port/esp32/netif/wlanif.c
+components/protocomm/include/security/protocomm_security.h
+components/protocomm/include/transports/protocomm_ble.h
+components/protocomm/src/common/protocomm.c
+components/protocomm/src/security/security1.c
+components/protocomm/src/transports/protocomm_nimble.c
+components/wifi_provisioning/include/wifi_provisioning/manager.h
+components/wifi_provisioning/include/wifi_provisioning/wifi_config.h
+components/wifi_provisioning/include/wifi_provisioning/wifi_scan.h
+components/wifi_provisioning/proto/wifi_config.proto
+components/wifi_provisioning/proto/wifi_scan.proto
+components/wifi_provisioning/proto-c/wifi_config.pb-c.c
+components/wifi_provisioning/proto-c/wifi_config.pb-c.h
+components/wifi_provisioning/proto-c/wifi_scan.pb-c.c
+components/wifi_provisioning/proto-c/wifi_scan.pb-c.h
+components/wifi_provisioning/python/wifi_config_pb2.py
+components/wifi_provisioning/python/wifi_constants_pb2.py
+components/wifi_provisioning/python/wifi_scan_pb2.py
+components/wifi_provisioning/src/manager.c
+components/wifi_provisioning/src/wifi_config.c
+components/wifi_provisioning/src/wifi_provisioning_priv.h
+components/wifi_provisioning/src/wifi_scan.c
+tools/idf_tools.py
+```
+
+## Feature List
+The following is a list of the SpinDance Embedded Starter Kit features that the changes in this fork support:
+- Network (NW) Metrics Reporting
+- Protocomm BLE Connectivity Reporting
+- Provisioning State in BLE Advertisement Data (AD)
+- JWT Authorization for Protocomm WiFi Provisioning
+- WPA2 Enterprise NW Support
+- WiFi Provisioning Sequence Changes
+- Python Version Change
+
+## Feature Details
+The features, their associated changes to ESP IDF and how the change is related to the Embedded Starter Kit are described in more detail below:
+- **Network (NW) Metrics Reporting**
+  - Added support for simple bytes in/bytes out tracking and reporting in ESP's LWIP implementation
+  - Embedded Starter Kit accesses these values via its PAL and includes them in metrics reported to MQTT
+  - Impacted ESP IDF files:
+    - wlanif.h
+    - wlanif.h
```

Comment:
accidental duplication or a cut-n-paste error in the name of the second file?

Reply:
Looking at the commit it's clear the second file should be `wlanif.c`. 
Will fix.